keras_nlp/models/bart/bart_seq_2_seq_lm.py

```diff
@@ -15,20 +15,31 @@
 
 import copy
 
-import tensorflow as tf
-
 from keras_nlp.api_export import keras_nlp_export
 from keras_nlp.backend import keras
+from keras_nlp.backend import ops
 from keras_nlp.models.bart.bart_backbone import BartBackbone
 from keras_nlp.models.bart.bart_presets import backbone_presets
 from keras_nlp.models.bart.bart_seq_2_seq_lm_preprocessor import (
     BartSeq2SeqLMPreprocessor,
 )
 from keras_nlp.models.generative_task import GenerativeTask
-from keras_nlp.utils.keras_utils import is_xla_compatible
 from keras_nlp.utils.python_utils import classproperty
 
 
+class ReverseEmbedding(keras.layers.Layer):
```

Comment:
It seems like this is repeated across a number of backbones -- maybe I'm missing some nuance but can it not be a shared component?

Reply:
I do want to do that eventually, but with a few more bells and whistles (there's a few different forms this embedding projection can take). I also have no idea what to call it when we do export it.

For now, I am working around the restriction that we can't apply `keras_core.ops` to `tf.keras` `KerasTensor`.

But I could just pop this unexported into a layer in `layers.modeling` until we figure out what to do with it? No strong opinion.

Reply:
Fine with me to keep it here for now then, or make it an __internal__ layer somewhere. But if it's going to move eventually, a TODO would be good